scripts/generate-git-snapshot

```diff
@@ -234,7 +234,7 @@ identify_latest_change() {
   fi
 
   echo "Latest tag [${last_tag:-}] / merge [${last_merge:-}] seems to be $since"
-  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS
+  ${GBP_DCH} -s "${since}" $DCH_COMBINED_OPTS $DCH_CHANGELOG_FILE
```

Comment:
Ah I see, that makes sense. Sure, please go for it, I'd happily accept the PR. Please provide this example in the git commit message so it's clear what's the use case and how to use it, thanks!